index.js

```diff
@@ -784,8 +807,11 @@ ShareDbMongo.prototype.skipPoll = function(collectionName, id, op, query) {
 
 function getFields(query) {
   var fields = {};
-  getInnerFields(query.$query, fields);
+  if (query.hasOwnProperty('$distinct')) {
```

Comment:
get rid of this. make skipPoll return false for any cursor operations or collection operations. leave comment that could be optimized further